vscode/src/commands/playCodio.ts

```diff
@@ -43,6 +47,16 @@ export default async function playCodio(
   }
 }
 
+/**
+ * Check if given object is a tree item.
+ * @param obj Object to check properties on.
+ * @return Return true if given object is a tree item; false other wise.
+ */
+function isTreeItem(obj = {}): boolean {
```

Comment:
isTreeItem sounds like a util, I would move it to a utils folder. The fact that it gets a codioUri, where the type is Uri means something should be changed in the type (and the name)

Reply:
It could get a codioUri but it is not determined until further object inspection. I will move this function into `utils.ts`.